internal/configs/annotations.go

```diff
@@ -66,7 +66,7 @@ var minionInheritanceList = map[string]bool{
 	"nginx.org/fail-timeout":             true,
 }
 
-func parseAnnotations(ingEx *IngressEx, baseCfgParams *ConfigParams, isPlus bool, hasAppProtect bool, enableInternalRoutes bool) ConfigParams {
+func parseAnnotations(ingEx *IngressEx, baseCfgParams *ConfigParams, isPlus bool, hasAppProtect bool, enableInternalRoutes bool, enableIngressSnippets bool) ConfigParams {
```

Comment:
should have unit tests for the same scenarios as well, like functional tests? It's be much quicker and will increase our existing coverage too, we should ideally have more unit tests than functional ones.